manifests/cli_helper.pp

```diff
@@ -25,12 +34,16 @@
   }
 
   # Provide the -i flag if specified by the user.
-  if $::jenkins::cli_ssh_keyfile {
-    $auth_arg = "-i ${::jenkins::cli_ssh_keyfile}"
+  if $ssh_keyfile {
+    $auth_arg = "-i ${ssh_keyfile}"
   } else {
     $auth_arg = undef
   }
 
+  if $ssh_keyfile != $::jenkins::cli_ssh_keyfile {
+    info("Using jenkins::cli_helper($ssh_keyfile) is deprecated and will be removed in the next major version of this module")
```

Comment:
`manifests/cli_helper.pp:44:variables_not_enclosed:WARNING:variable not enclosed in {}`